cpp/src/cider/exec/nextgen/jitlib/llvmjit/LLVMJITEngine.h

```diff
@@ -31,9 +31,9 @@ struct CompilationOptions {
   bool optimize_ir = true;
   bool aggressive_jit_compile = true;
   bool dump_ir = false;
-  bool enable_vectorize = false;
+  bool enable_vectorize = true;
   bool enable_avx2 = true;
-  bool enable_avx512 = false;
+  bool enable_avx512 = true;
```

Comment:
Maybe AVX512 shouldn't be enabled, seems it has no benefit to performance now.

Reply:
I'll try to test on icx, if nothing gains, I'll keep it closed.

Reply:
In IceLake, it has no gains. Not sure about SPR.

Reply:
no performance gains for the small batch. for large batch,  it depends. 
keep it closed